### PR TITLE
[MWPW-121124] Updates to split marquee

### PR DESCRIPTION
**Pull request by elan-tbx**
* Completed feature requests for additional split `Marquee` variants, including:
  * A 1/3 split sub-variant
  * A `row-reversed` option for the split layout
  * A media credit area
  * Refined guardrails on the icon area
  * Aligned styles with Consonant designs

Resolves: [MWPW-121124](https://jira.corp.adobe.com/browse/MWPW-121124)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/drafts/ebartholomew/split-marquee?martech=off
- After: https://split-marquee--milo--adobecom.hlx.page/drafts/ebartholomew/split-marquee?martech=off

### CODE REVIEW

**Review by vhargrave — COMMENTED** [review]
Looks pretty good ! 
I'd add a small unit test for the javascript code you added to get our test coverage back up.

**Comment by elan-tbx** [comment]
@vhargrave thanks! Should be good to go now

**Review by vhargrave — APPROVED** [review]
looking good!

File: libs/blocks/marquee/marquee.css [hunk 1]
```diff
@@ -189,10 +216,23 @@
   margin-bottom: var(--spacing-s);
 }
 
+.marquee.split a:not(.con-button) {
+  color: var(--color-accent);
+  text-decoration: underline;
+}
```
Comment (ryanmparrish) [inline libs/blocks/marquee/marquee.css:222]: This seems over reaching. We probably want to rely on site defaults for general link color/underline behavior. If those need to change? we should do it in styles/variables.css globally and make this a separate task.

File: libs/blocks/marquee/marquee.css [hunk 2]
```diff
@@ -189,10 +216,23 @@
...
+
+.marquee.split .button-justified-mobile {
+  width: 100%;
```
Comment (ryanmparrish) [inline libs/blocks/marquee/marquee.css:225]: Just wanted to check if this should be updated at the typography.css level instead? Is this needed?

**Review by ryanmparrish — CHANGES_REQUESTED** [review]

File: libs/blocks/marquee/marquee.js [hunk 3]
```diff
@@ -108,5 +112,13 @@ export default function init(el) {
       media.classList.add('bleed');
       foreground.insertAdjacentElement('beforebegin', media);
     }
+    if (media?.lastChild.textContent.trim()) {
+      const mediaCredit = document.createElement('div');
```
[the remark this answers is not in the document]
Comment (elan-tbx) [reply libs/blocks/marquee/marquee.js]: Alright, updated to use createTag. FYI there is test coverage for this on line 34 of marquee.test.js

File: libs/blocks/marquee/marquee.css (hunk 1 above)
Comment (elan-tbx) [reply libs/blocks/marquee/marquee.css]: @ryanmparrish  Makes sense to me. I've removed this rule, but this does make the simple links [look incorrect now](https://split-marquee--milo--adobecom.hlx.page/drafts/ebartholomew/split-marquee?martech=off). Should we create the separate task for that styling and address that before continuing with this PR?

File: libs/blocks/marquee/marquee.css (hunk 2 above)
Comment (elan-tbx) [reply libs/blocks/marquee/marquee.css]: Good point, adjusted my styles so they adjust flexbox rules accordingly

**Review by ryanmparrish — APPROVED** [review]
Looking good!

**Pull request merged**